WMF Framework/Theme.swift

```diff
@@ -169,580 +93,714 @@ public extension UIColor {
-        self.hintBackground = hintBackground
-        self.hintWarningBackground = hintWarningBackground
-        self.animationBackground = animationBackground
-        self.referenceHighlightBackground = referenceHighlightBackground
-        
-        self.overlayText = overlayText
-        
-        self.searchFieldBackground = searchFieldBackground
-        self.keyboardBarSearchFieldBackground = keyboardBarSearchFieldBackground
-        
-        self.primaryText = primaryText
-        self.secondaryText = secondaryText
-        self.tertiaryText = tertiaryText
-        self.disabledText = disabledText
-        self.disabledLink = disabledLink
-        
-        self.chromeText = chromeText
-        
-        self.link = link
-        self.accent = accent
-        
-        self.border = border
-        self.shadow = shadow
-        
-        self.icon = icon
-        self.iconBackground = iconBackground
-        
-        self.rankGradientStart = rankGradientStart
-        self.rankGradientEnd = rankGradientEnd
-        self.rankGradient = Gradient(startColor: rankGradientStart, endColor: rankGradientEnd)
-        
-        self.error = error
-        self.warning = warning
-        self.destructive = destructive
-        self.secondaryAction = secondaryAction
-        self.unselected = unselected
-        
-        self.blurEffectStyle = blurEffectStyle
-        self.blurEffectBackground = blurEffectBackground
-        
-        self.tagText = tagText
-        self.tagBackground = tagBackground
-        self.tagSelectedBackground = tagSelectedBackground
-        
-        self.distanceBorder = distanceBorder
-        self.descriptionBackground = descriptionBackground
-        self.descriptionWarning = descriptionWarning
-        
-        self.inputAccessoryBackground = inputAccessoryBackground
-        self.inputAccessoryButtonTint = inputAccessoryButtonTint
-        
-        self.pageIndicator = pageIndicator
-        self.pageIndicatorCurrent = pageIndicatorCurrent
-        
-        self.unreadIndicator = unreadIndicator
-        self.depthMarker = depthMarker
-        
-        self.diffTextAdd = diffTextAdd
-        self.diffTextDelete = diffTextDelete
-        self.diffHighlightAdd = diffHighlightAdd
-        self.diffHighlightDelete = diffHighlightDelete
-        self.diffStrikethroughColor = diffStrikethroughColor
-        self.diffContextItemBackground = diffContextItemBackground
-        self.diffContextItemBorder = diffContextItemBorder
-        self.diffMoveParagraphBackground = diffMoveParagraphBackground
-        self.diffCompareAccent = diffCompareAccent
-        self.diffCompareChangeHeading = diffCompareChangeHeading
-        self.talkPageBackground  = talkPageBackground
-        self.talkPageCoffeRollBackground = talkPageCoffeRollBackground
+    public var diffTextAdd: UIColor {
+        switch identifier {
+        case .light:
+            return .green100
```

Comment:
Seems like these are off from the spreadsheet, should be `.gray700` for light, and `green600` for the others.